LinkedList/AddTwoLinkedList.py

```
@@ -0,0 +1,62 @@
+              l2=l2.next 
+          if carry==1:
+              self.insert(carry)
+          return self.reverse(self.head) 
+                
+                
+      def reverse(self,linkedList:ListNode):
+          
+          prev=None
+          temp=linkedList
+          while temp:
+                next=temp.next 
```

Pylint (3.3.7)

[convention] Trailing whitespace

(C0303)